SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/sh
+#!/bin/bash
 
 # Description: This script is designed to mount a DM-Verity root filesystem and
-# set up an OverlayFS. It is driven by kernel parameters and is invoked during
-# the dracut initramfs phase.
+# set up OverlayFS. It is driven by kernel parameters and is invoked during the
```

Comment:
Can you #-mention the `3.0-dev` PR you're cherry-picking here? It would make the review easier.

Reply:
I feel like I will include them in the PR description instead since the info is not related to the script itself.